flaml/nlp/autotransformers.py

```diff
@@ -74,9 +78,14 @@ def _set_search_space(self,
                           **custom_hpo_args):
         from .hpo.hpo_searchspace import AutoHPOSearchSpace
 
+        if self.jobid_config.spa == "grid":
+            model_type = "bert"
```

Comment:
Does 'grid' only work for bert?
Please rename `.spa` to more informative name

Reply:
"Does 'grid' only work for bert?" => fixed. 

"Please rename .spa to more informative name" => spa is the abbreviated name for space_mode. The abbreviated name is only for logging in azure, not meant to be informative. The informative version for user is defined in nlp/utils.py

Reply:
not informative for developers either. rename for code readability and maintainability.